refactor/modules/numeric_hash.py

```diff
@@ -0,0 +1,31 @@
+import math
+import numpy as np
+
+
+class Numerichash:
+    def __init__(self, r, b):
+        self.num_rows = r
+        self.num_buckets = b
+        self.count = np.zeros((self.num_rows, self.num_buckets))
+
+    def hash(self, cur_node, hacked_lsh, max, min):
```

Comment:
Do we still need `hacked_lsh, max, min`? Doesn't seem to be in use

Reply:
It is a remnant of our attempt to design an ideal hash. If you look at numerichash.cpp in the hash function you will see this commented block:
```
//TO DO - Check if we want to use the flag for hacked lsh and modify hash accordingly
    /*if (hacked_lsh == 1) {
        bucket = cur_node/((max-min)/num_buckets);
        cout << "bucket reeturned for " << cur_node << "\n";
        cout << bucket << "\n";
    } else {
        cur_node = cur_node * (num_buckets - 1);
        bucket = floor(cur_node);
        if(bucket < 0)
            bucket = (bucket%num_buckets + num_buckets)%num_buckets;
    }*/
```
I kept the params for convenience but didn't implement the "hacked lsh" in python because we were not sure how beneficial it is... 
let's discuss this in more detail tomorrow :)